### PR TITLE
Update controlling-your-data-using-customer-key.md

### PR DESCRIPTION
**Pull request by MikePlumleyMSFT**
Multi-geo is generally available.

### CODE REVIEW

**Comment by mypil** [comment]
@KCCross - PR is ready for final review.  

Thank you.

cc: @kenwith

File: microsoft-365/compliance/controlling-your-data-using-customer-key.md [hunk 1]
```diff
@@ -499,7 +499,7 @@ To set up Customer Key for SharePoint Online and OneDrive for Business, you will
 ### Create a data encryption policy (DEP) for each SharePoint Online and OneDrive for Business geo
```
Comment (imbadjoe1993) [inline microsoft-365/compliance/controlling-your-data-using-customer-key.md:499]: What does this mean

**Review by imbadjoe1993 — CHANGES_REQUESTED** [review]

**Comment by imbadjoe1993** [comment]
#523

**Comment by KCCross** [comment]
Thanks for the update. We are about to publish a total overhaul of this content. I've rolled in the feedback from this PR into that rewrite so I'm closing this PR. Thanks everyone!

**Pull request closed without merging**